hicommonwealth/commonwealth
# Add group stats to admin panel

## Link to Issue
Closes: #6179

## Description of Changes
- Consolidates the `/communities/:communityId/stats` route into the `/admin/analytics` route so it supports both global and per-community stats (same response data structure)
  - Removes redundant code
- Moves server business logic into new Admin controller
- Updates UI to reflect API change

## Test Plan
- Go to admin panel: http://localhost:8080/admin-panel
- Confirm that global stats show on page load
- Under "Community lookup", type a community that you've been active in recently (within the last month)
  - Confirm that the stats reflect the recent activity

## Deployment Plan
N/A

## Other Considerations
N/A
